draft-ietf-rats-ar4si.md

```diff
@@ -227,7 +230,7 @@ After the Verifier performs its appraisal, it will include sufficient informatio
 
 This document recognizes three general categories of Attesters.
 
-1. HSM-based: A Hardware Security Module (HSM) based cryptoprocessor which continually hashes security measurements in a way which prevents an Attester from lying about measurements which have been extended into the Attesting Environment (e.g., TPM2.0.)
+1. HSM-based: A Hardware Security Module (HSM) based cryptoprocessor which hashes one or more streams of security measurements from an Attester within the Attesting Environment. Maintenance of this hash enables detection of an Attester which is lying about the set of security measurements taken. An example of a HSM is a TPM2.0 {{TPM2.0}}.
 2. Process-based: An individual process which has its runtime memory encrypted by an Attesting Environment in a way that no other processes can read and decrypt that memory (e.g., {{SGX}} or {{-PSA}}.)
```

Comment:
Out of scope note re: points 2 and 3.  The text seems overly prescriptive: encryption is one (but not the only) technique to achieve memory isolation.